index.php

```diff
@@ -19,9 +24,53 @@
 
             <input type="search" class="input-search" placeholder="Search Here">
 
+
         </header>
+
+        <section class="content">
+
+            <nav>
+                <ul class="navbar">
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#seasonal" class="active">Seasonal</a></li>
+                    <li><a href="#category">Category</a></li>
+
+                </ul>
+            </nav>
+
+            <div class="fruit-definition">
+                <h1>Pineapple</h1>
+                <p>The pineapple is a tropical plant with an edible fruit, also called a pineapple and the most economically significant plant in the family Bromeliaceae.</p>
+
+            </div>
+
+            <div class="center_image">
```

Comment:
This class is also using `_` instead of `-`, Also I think we can rename this class and use something more specific, for example: `fruit-image`